talent/job-descriptions/perks-section.md

```diff
@@ -1,9 +1,9 @@
 ## Perks
  
- - We go to conferences and other developer events ! ;)
- - Open Source Days, 10% of the time you can work on any OSS project you choose.
- - Flexible hours and the possibility of remote work.
- - Free books. Company will buy any books that can help you be even better at your work.
- - Spacious physical office, close to public transit and to the Retiro park.
- - Monthly get-togethers, annual summer and winter Xmas parties, a hackathon retreat.
- - Our own, open source craft beers https://github.com/src-d/homebrew.
\ No newline at end of file
+ - We go to conferences and other developer events!
+ - Open Source Days, every second Monday, you are encouraged to work on any OSS project you choose.
+ - Flexible hours, set your own schedule that fits you.
+ - Free books. We will buy any books that help you learn & grow.
+ - If you choose to work from one of our offices, you will enjoy a light and spacious environment
```

Comment:
light is ambiguous on this context: do you mean well illuminated?
maybe use comfortable and spacious?